src/DataWrangling/metadata.jl

```diff
@@ -0,0 +1,113 @@
+using CFTime
+using Dates
+using Base: @propagate_inbounds
+
+struct Metadata{D, V} 
+    name  :: Symbol
+    dates :: D
```

Comment:
I see `dates` here but elsewhere `times`. Pick one...

Reply:
dates it better, times has no reference